README.md

```diff
@@ -7,7 +7,7 @@ This is [Truss](https://truss.works/)' custom-built docker image for use with Ci
 The following languages are installed:
 
 * Python 3.7.x (container base image)
-* Go 1.12.x
+* Go 1.12.9
```

Comment:
I think this was left generic on purpose. So as to say "We support 1.12 and no promises on the patch version" for whoever is using it. We update the patch frequently enough that its just a lot of churn in this file.

Reply:
Ok - I've removed it. Thanks for the catch!